pages/common/git-remote.md

```diff
@@ -0,0 +1,23 @@
+#git remote
+
+> Manage set of tracked repositories (“remotes”)
+
+- Show a list of existing remotes, their names and URL
+
+`git remote -v`
+
+- Add a remote
+
+`git remote add {{remote_name}} {{remote_url}}`
+
+- Add a remote and run fetch on it
```

Comment:
is this really such a common use case? I'd suggest instead having the command to change the URL of a remote, which I personally have had to do more than once (e.g. after forgetting to make a clone of a github repo in the web interface before cloning to my machine, so I could push to it).